applications/sasquatch/charts/scimma/templates/kafka-topics.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaTopic
+metadata:
+  name: "lsst.scimma.too_alert"
```

Comment:
The use of  `_` or  `-` as separator is a little problematic. Strimzi does not accept `_` in the topic name, and if you use `-` you have to be careful when creating the Avro schema because the REST Proxy doesn't like `-` in the schema name. My preference here would be camel case for the topic and schema names `lsst.scimma.tooAlert`